docs/application/dotnet/guides/nui/styling-controls-with-JSON.md

```diff
@@ -230,6 +230,36 @@ You can implement transitions in two ways:
     ]
     ```
 
+## Apply JSON style
+
+Style can be applied at runtime using the **StyleManager** API, as shown in the following example code:
```

Comment:
Strictly StyleManager is a class. So I suggest to remove the "API". How about below?

To apply Style on runtime, use **StyleManager** as the following code:

Reply:
Done.

Reply:
@mijin85cho slight modification to maintain consistency:
```suggestion
To apply Style at runtime, use the **StyleManager** class as follows:

Reply:
@nawab-samsung I think your suggestion is acceptable.

Reply:
@mihashco request you to incorporate the changes.

Reply:
Done